collect_app/src/androidTest/java/org/odk/collect/android/dao/FormsDaoTest.java

```diff
@@ -126,7 +128,7 @@ public void getFormsCursorTest() {
 
     @Test
     public void getFormsCursorForFormFilePathTest() {
-        Cursor cursor = formsDao.getFormsCursorForFormFilePath(Collect.FORMS_PATH + "/Miramare.xml");
+        Cursor cursor = formsDao.getFormsCursorForFormFilePath(storagePathProvider.getDirPath(StorageSubdirectory.FORMS) + "/Miramare.xml");
```

Comment:
This is going to do a db request. Shouldn't it use the `getDbPath` method? As written, this code will always build an absolute path but when we flip the switch to scoped storage, there won't be absolute paths in the db to query against.

Reply:
getFormsCursorForFormFilePath() used here deals with it.